tests/snapshots/assets__check_file@packages-fletcher-diagram.typ-40.snap

```diff
@@ -670,17 +674,19 @@ snapshot_kind: text
         size: node.size,
       ))
 
-    let vertices-affecting-grid = edges
-      .map(edge => {
-        resolve-edge-vertices(
-          edge,
-          ctx: ctx-with-uv-anchors + (
-            target-system: "uv",
-          ),
-          nodes,
-        )
-      })
-      .join() + () // coerce none to ()
+    let vertices-affecting-grid = (
+      edges
+        .map(edge => {
+          resolve-edge-vertices(
+            edge,
+            ctx: ctx-with-uv-anchors
+              + (target-system: "uv"),
+            nodes,
+          )
+        })
+        .join()
+        + ()
+    ) // coerce none to ()
```

Comment:
We add parens here. That's interesting.. Is it needed?

Reply:
![image](https://github.com/user-attachments/assets/4a763f46-cade-47f5-9188-c6434eaea8dc)

Reply:
We are adding excessive parens here

Reply:
It's necessary only when we decide to put `+` on a separate line.